src/main/java/org/ambraproject/wombat/service/PeerReviewServiceImpl.java

```diff
@@ -174,7 +207,29 @@ String getReviewXml(Map<String, ?> metadata) throws IOException {
 
   String getContent(ContentKey contentKey) throws IOException {
     CloseableHttpResponse response = corpusContentApi.request(contentKey, ImmutableList.of());
-    String letterContent = EntityUtils.toString(response.getEntity(), "UTF-8");
-    return letterContent;
+    String content = EntityUtils.toString(response.getEntity(), "UTF-8");
+    return content;
+  }
+
+  /**
+   * Extract received date from article xml.
+   * @param xml article xml
+   * @return received date.
+   */
+  String getReceivedDate(String xml) throws IOException {
+
+    String receiveDateXml = XmlUtil.extractElement(xml, "date", "date-type", "received");
+
+    String receivedDateString = String.format("%s %s %s",
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "month")),
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "day")),
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "year")));
+
+    DateTimeFormatter parseFormatter   = DateTimeFormatter.ofPattern("M d yyyy");
+    DateTimeFormatter displayFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy");
```

Comment:
You might want to move this closer to where it's used.

Reply:
Moved.